src/hooks/useScrollFlash.ts

```diff
@@ -0,0 +1,27 @@
+import React, {useEffect} from 'react'
+/**
+ * This hook will flash the scrollbars for a ref of a container that has scrollable overflow
+ * @param scrollContainerRef The ref of the scrollable content
+ */
+export default function useScrollFlash(scrollContainerRef: React.RefObject<HTMLElement>) {
+  // https://adxlv.computer/projects/flash-scrollers/
+  useEffect(() => {
+    const scrollContainer = scrollContainerRef.current
+    if (!scrollContainer) {
+      return
+    }
+    const currentScroll = scrollContainer.scrollTop
+    const maxScroll = scrollContainer.scrollHeight
+
+    const altScroll = (() => {
+      if (currentScroll < 1 || currentScroll < maxScroll) return currentScroll + 1
+      else return currentScroll - 1
+    })()
+
+    scrollContainer.scrollTop = altScroll
+    scrollContainer.scrollTop = currentScroll
+
+    const flashScrollersEvent = new Event('flashScrollers')
```

Comment:
Are we listening for this event somewhere?

Reply:
Resolved in https://github.com/primer/components/pull/1276/commits/bc4b4b1ba4bdcbe5240de7216c9db172901ca5ab